### PR TITLE
Add missing ESPHome for home automaction

### PR DESCRIPTION
**Pull request by cooljimy84**
Hope this helps as i use ESPHome more than tasmota on my devices

### CODE REVIEW

**Review by RealOrangeOne — DISMISSED** [review]

File: docs/home-automation/ESPHome/index.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# ESPHome
+
+ESPHome is a system to control your ESP8266/ESP32 by simple yet powerful configuration files and control them remotely through Home Automation systems.
+## Introduction
```
Comment (py-crash) [inline docs/home-automation/ESPHome/index.md:4]: White line missing

File: docs/home-automation/ESPHome/index.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+* [Converting from Tuya firmware](https://esphome.io/guides/migrate_sonoff_tasmota.html)
+* [ESPHome template for devices](https://esphome.io/#cookbook)
+* [ESPHome DIY Examples](https://esphome.io/guides/diy.html)
+
+## Installation
+The main ways of installing ESPHome on a device are:
+
+* OTA (over the air) via tuya convert (see link above)
+* Flashing over serial.
```
Comment (py-crash) [inline docs/home-automation/ESPHome/index.md:21]: Reapeated information

**Review by py-crash — CHANGES_REQUESTED** [review]

**Comment by cooljimy84** [comment]
Sorry, bit of a Github noob here,

**Comment by ironicbadger** [comment]
#118

**Pull request closed without merging**